composer.json

```diff
@@ -110,7 +110,7 @@
         "scssphp/scssphp": "v1.12.0",
         "setasign/fpdi": "^2.3.7",
         "setasign/tfpdf": "^1.33",
-        "shopware/conflicts": ">=0.3.0",
+        "shopware/conflicts": "dev-NEXT-40114/update-twig",
```

Comment:
we need a new conflicts version for that right?

Reply:
Yes